src/Translatable/Entity/Translation.php

```diff
@@ -9,40 +9,37 @@
 
 namespace Gedmo\Translatable\Entity;
 
-use Doctrine\ORM\Mapping\Entity;
-use Doctrine\ORM\Mapping\Index;
-use Doctrine\ORM\Mapping\Table;
-use Doctrine\ORM\Mapping\UniqueConstraint;
+use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Translatable\Entity\MappedSuperclass\AbstractTranslation;
 use Gedmo\Translatable\Entity\Repository\TranslationRepository;
 
 /**
  * Gedmo\Translatable\Entity\Translation
  *
- * @Table(
+ * @ORM\Table(
  *     name="ext_translations",
  *     options={"row_format": "DYNAMIC"},
  *     indexes={
  *
```

Comment:
```suggestion
```
According to https://github.com/doctrine-extensions/DoctrineExtensions/actions/runs/6693389572/job/18194664080?pr=2703

Reply:
Fixed it, somehow came in by the rebasing before

Reply:
Yep, out of curiosity I guess it was because this: 

https://github.com/doctrine-extensions/DoctrineExtensions/blob/058e95a34847b32d57b5583a006911a5465dac3f/.php-cs-fixer.dist.php#L62-L66

before this change they weren't in those groups